css/main.css

```diff
@@ -0,0 +1,1623 @@
+  background-size: cover;
+}
+
+.f3-background-9 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/acquisition-blog-post-thisstartupannouncementdoesntexist-368x276.png");
+  background-size: cover;
+}
+
+.f3-img-text {
+  text-decoration: none;
+  line-height: 25px;
+  position: relative;
+  top: 150px;
+  padding-left: 6px;
+}
+
+.linear-gradient {
+  background: rgba(250, 0, 0, 0.25) url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/image-gradient.png") no-repeat;
+  background-size: 100% 100%;
+  height: 200px;
+}
+
+.linear-gradient-1 {
+  background: transparent url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/image-gradient.png") no-repeat;
+  background-size: 100% 100%;
+  height: 278px;
+}
+
+.linear-gradient-2 {
+  background: transparent url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/image-gradient.png") no-repeat;
+  background-size: 100% 100%;
+  height: 270px;
+}
+
+.linear-gradient-3 {
+  background: transparent url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/image-gradient.png") no-repeat;
+  background-size: 100% 100%;
+  height: 200px;
+}
+
+.f3-img-text h3 {
+  margin: 0;
+  font-size: 12px;
+}
+
+.f3-text {
+  padding: 0 0 0 0;
+  font-size: 10px;
+}
+
+.special {
+  color: rgb(170, 168, 168) !important;
+}
+
+.top {
+  color: rgb(209, 206, 206);
+  font-size: 14px;
+}
+
+.middle {
+  color: white;
+}
+
+.bottom {
+  font-size: 10px;
+  font-weight: 500;
+  color: rgb(255, 247, 247);
+}
+
+.f3-second {
+  width: 100%;
+  height: 75px;
+  background-color: rgb(253, 221, 221);
+  position: relative;
+  top: -23px;
+}
+
+.f3-title-size {
+  width: 244px;
+  height: 19px;
+  margin: 0;
+  color: rgb(255, 68, 34);
+  margin-top: -20px;
+  font-size: 12px;
+}
+
+.f3-text-size {
+  text-decoration: none;
+  width: 244px;
+  height: 50px;
+  margin-right: 0;
+  font-size: 10px;
+  color: rgb(255, 68, 34);
+  line-height: 5px;
+  text-align: left;
+}
+
+.f3-img {
+  float: right;
+  width: 75px;
+  height: 75px;
+  position: relative;
+  top: -55px;
+  right: 0;
+}
+
+.f3-third {
+  width: 100%;
+  height: auto;
+  background-color: rgb(253, 253, 253);
+  position: relative;
+  top: -20px;
+}
+
+.f3-third ul {
+  list-style-type: none;
+  width: 90%;
+  margin: 0;
+  padding-left: 17px;
+}
+
+.f3-third ul li::before {
+  content: "\2022";
+  color: rgb(255, 68, 34);
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+}
+
+.f3-list {
+  width: 100%;
+  font-size: 10px;
+  padding-bottom: 5px;
+  height: 50px;
+  line-height: 19px;
+  text-align: left;
+  border-bottom: 1px solid black;
+}
+
+.section-4 {
+  width: auto;
+  height: 660px;
+  margin: 0;
+  padding: 0;
+}
+
+.f4-grid {
+  display: block;
+}
+
+.f4 {
+  width: 294px;
+  height: 298px;
+  border: 1px solid transparent;
+}
```

Comment:
- [ ] Here notice that the alignment is totally different, you can change your .f4 here to this:
```
.f4 {
    display: flex;
    border: 1px solid transparent;
}
```

Then change the image size. And use display: none; for the yellow label